public/video-ui/src/components/VideoData/VideoData.js

```diff
@@ -125,7 +125,7 @@ class VideoData extends React.Component {
             </ManagedField>
           </ManagedSection>
           <ManagedSection>
-            <ManagedField fieldLocation="expiryDate" name="Expiry Date">
+            <ManagedField fieldLocation="contentChangeDetails.expiry.date" name="Expiry Date">
```

Comment:
I don't think this'll work for older atoms as this field will be empty until we re-index.